src/storage/loader_uadetrac.py

```diff
@@ -15,27 +15,32 @@
 import cv2
```

Comment:
we can't afford to keep separate loader for each dataset. have a look at abstractVideoLoader/ SImpleVideoLoader. we might have to replace this or should we even need this.